pkg/controller/machineset/controller.go

```diff
@@ -109,13 +110,7 @@ type ReconcileMachineSet struct {
 
 func (r *ReconcileMachineSet) MachineToMachineSets(o handler.MapObject) []reconcile.Request {
 	result := []reconcile.Request{}
-	m := &clusterv1alpha1.Machine{}
-	key := client.ObjectKey{Namespace: o.Meta.GetNamespace(), Name: o.Meta.GetName()}
-	err := r.Client.Get(context.Background(), key, m)
-	if err != nil {
-		klog.Errorf("Unable to retrieve Machine %v from store: %v", key, err)
-		return nil
-	}
+	m := (*clusterv1alpha1.Machine)(unsafe.Pointer(&o.Object))
```

Comment:
o_O

Reply:
what's the unsafe for

Reply:
it's trying to get the pointer of the machine object and to be used later...

Reply:
Could we achieve the same thing with
```go
m, ok := o.Object.(*clusterv1alpha1.Machine)
```
?